contractcourt/breacharbiter.go

```diff
@@ -1705,6 +1755,8 @@ func (bo *breachedOutput) Encode(w io.Writer) error {
 		return err
 	}
 
+	// TODO(roasbeef): need to write taptweak...
```

Comment:
needs to be written in case of restart?

Reply:
Yep, although technically if we have the control block, and the revealed script, then we can re-derive the tweak. I might go that direction instead of adding more state here.